docs/website/data/integrations.yaml

```diff
@@ -1,6 +1,20 @@
 # This file is used to generate the OPA ecosystem page: https://www.openpolicyagent.org/docs/latest/ecosystem/
 # For instructions on how to add an entry, see https://github.com/open-policy-agent/opa/tree/main/docs#integration-index
 integrations:
+  apache-apisix:
+    title: Authorization Integration with Apache APISIX
+    description: Apache APISIX is a dynamic, real-time, high-performance API gateway. Integration of Apache APISIX and OPA can provide user with more secure API authorizations.
+    software:
+      - Apache APISIX
+    lables:
+      category: gateway
+      layer: network
+    # tutorials:
```

Comment:
You can remove the commented lines here. We can add tutorials and blogs later :)